backend/src/documentation/padding-oracle.md

```diff
@@ -0,0 +1,171 @@
+# Padding Oracle Attack
+
+For this section is important that you read and understand the [XOR docs](/docs/xor). 
+
+
+## The Basics
+
+Firstly, to understand the _padding oracle attack_ also known as _Vaudenay attack_ it is important to know clearly the meaning of **padding** and **oracle**. 
+
+### Padding
+
+The CBC mode encrypts blocks with the same block length. As you may think, certainly not all the plaintext messages length are a multiple of the cipher block length. So, how this problem can be solved? How a 14-byte message is encrypted where cipher blocks are 8 bytes? One of the techniques used is **padding**.
+
+The **padding** makes the ciphertext longer than the plaintext to maintain each block with the same size allowing to encrypt a message of any length. For block ciphers, padding is applied according to the [PKCS#7](https://en.wikipedia.org/wiki/Padding_(cryptography)#PKCS#5_and_PKCS#7) standard and [RFC 5652](https://tools.ietf.org/html/rfc5652#section-6.3). The standard says, the value to pad with is the number of bytes of padding that remains to fill a block. More technically, the message is expanded and extra bytes are added to the plaintext in order to complete a block. 
```

Comment:
i wouldn't say the padding makes the ciphertext longer, as it only happens if the padding is the number of bytes in the block. otherwise is making the plaintext longer, which will result in a ciphertext with the expected size, the same if it has 3 byte padding or 4byte padding